Formula/ipopt.rb

```diff
@@ -0,0 +1,50 @@
+class Ipopt < Formula
+  desc "Interior point optimizer"
+  homepage "https://projects.coin-or.org/Ipopt/"
+  url "https://www.coin-or.org/download/source/Ipopt/Ipopt-3.12.12.tgz"
+  sha256 "7baeb713ef8d1999bed397b938e9654b38ad536406634384455372dd7e4ed61f"
+  head "https://github.com/coin-or/Ipopt.git"
+
+  depends_on "gcc"
+
+  def install
+    ENV.delete("MPICC")
+    ENV.delete("MPICXX")
+    ENV.delete("MPIFC")
+    ENV.deparallelize
+    ENV.fortran
+
+    chdir buildpath/"ThirdParty/Mumps" do
+      system "./get.Mumps"
```

Comment:
This is not how we handle dependencies in Homebrew. You will need to provide `mumps` as a separate formula, or to vendor it inside this formula (as a `resource` block).

Reply:
I know, but the MUMPS Makefile is in such bad shape, there is not a responsive upstream, and the IPOPT mangles the library name so it is not going to collide. Basically, this is going to be the only way to get IPOPT into Homebrew (search closed pull requests for a refresher).

Reply:
Would using a resource block to get MUMPS and the build files in IPOPT be acceptable?

Reply:
Let's try it that way, yes.

Reply:
Done.